### PR TITLE
⚠ Move all packages from /pkg to /internal

### PR DESCRIPTION
**Pull request by kevinrizza**
Ahead of an initial 1.0.0 release, we shouldn't preemptively expose any packages publicly. Instead, let's start my making any packages internal, and we can expose them publicly later on. This commit moves everything currently in /pkg to /internal for now.

### CODE REVIEW

**Comment by tmshort** [comment]
We have a bunch of other commits (i.e. @dtfranz's import of rukpak) that we may want to wait on before we do this.

**Comment by tmshort** [comment]
@kevinrizza, this needs to be rebased as #1032 was merged

**Comment by tmshort** [comment]
@kevinrizza are you going update this?

File: cmd/manager/main.go [hunk 1]
```diff
@@ -57,8 +59,6 @@ import (
 	"github.com/operator-framework/operator-controller/internal/rukpak/preflights/crdupgradesafety"
 	"github.com/operator-framework/operator-controller/internal/rukpak/source"
 	"github.com/operator-framework/operator-controller/internal/version"
-	"github.com/operator-framework/operator-controller/pkg/features"
-	"github.com/operator-framework/operator-controller/pkg/scheme"
```
Comment (tmshort) [inline cmd/manager/main.go:61]: The location of these shouldn't have changed. This will likely fail lint.

Comment (perdasilva) [reply cmd/manager/main.go:61]: fixed!

**Review by tmshort — APPROVED** [review]
/lgtm
API diff is expected as components are being moved from `pkg` to `internal`

**Pull request merged**